### PR TITLE
NETOBSERV-1214 & NETOBSERV-1399 Cluster name & zone features

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

- [x] Filter unecessary columns from operator
- [x] Specify `MultiClusterDeployment` and `AddZone` states to frontend `features` array

## Dependencies

https://github.com/netobserv/network-observability-console-plugin/pull/462

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: pkg/loki/labels.go
Comment (jotak) [inline pkg/loki/labels.go:23]: after rebasing you could use that:
```suggestion
	if helper.IsZoneEnabled(&desired.Processor) {
```

File: controllers/consoleplugin/consoleplugin_objects.go
Comment (jotak) [inline controllers/consoleplugin/consoleplugin_objects.go:416]: we could also add to columns config something like a `requiresFeature` field, that could be managed either here or on the frontend, to avoid having specific code for each feature here to skip columns.
E.g:

```yaml
  - id: ClusterName
    name: Cluster
    tooltip: The cluster ID or Name.
    field: K8S_ClusterName
    filter: cluster_name
    default: false
    width: 15
    requiresFeature: multiCluster
```

Comment (jpinsonneau) [reply controllers/consoleplugin/consoleplugin_objects.go:416]: Yes I'm fine with that

**Comment by jpinsonneau** [comment]
@jotak I rebased and addressed your comments. I'm updating the console plugin accordingly

**Review by OlivierCazade — APPROVED** [review]
LGTM

**Comment by jotak** [comment]
@jpinsonneau I guess we can merge it?

**Comment by jpinsonneau** [comment]
> @jpinsonneau I guess we can merge it?

I'm waiting for QE review before merging, just in case I need to update backports and so on.
Is it blocking us on something else ?

**Comment by jpinsonneau** [comment]
Resolved conflict https://github.com/netobserv/network-observability-operator/pull/546/commits/b1b8da0a309cb70360489a0918e410364280663f

**Pull request merged**
